### PR TITLE
[eslint] fix 'Modernizr' is not defined  no-undef

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Solves eslint errors for no-undef and useful for removing the off found in this PR: https://github.com/aces/Loris/pull/7137

Fixes 'Modernizr' is not defined  no-undef

### CODE REVIEW

**Comment by laemtl** [comment]
@maltheism eslint is not passing :)

**Review by laemtl — DISMISSED** [review]
@maltheism eslint is not passing :)

**Comment by maltheism** [comment]
Hi @laemtl see latest commit and I'm open for any suggestions.

**Comment by cmadjar** [comment]
@driusan could you review this?

File: modules/api_docs/jsx/swagger-ui_custom.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import 'swagger-ui-dist/swagger-ui.css';
 
 window.addEventListener('load', () => {
   const ui = new SwaggerUIBundle({
+    // eslint-disable-next-line no-undef
```
Comment (driusan) [inline modules/api_docs/jsx/swagger-ui_custom.js:6]: why is this here?

Comment (maltheism) [reply modules/api_docs/jsx/swagger-ui_custom.js:6]: Hi @driusan the variable "schema_urls" is undefined and created in php. So the eslint-disable-next-line no-def is one solution. @laemtl do you know if there's a more elegant solution for this?

Comment (driusan) [reply modules/api_docs/jsx/swagger-ui_custom.js:6]: I still don't understand.. how is php defining things in the javascript file? Is it a global variable? If so, can it be explicitly defined as one somewhere?

Comment (maltheism) [reply modules/api_docs/jsx/swagger-ui_custom.js]: @driusan I meant the php line `$this->tpl_data['schema_urls'] = json_encode($api_specs);`. I think the global variable solution is best as well. I added the change in latest commit.

File: modules/media/jsx/editForm.js [hunk 2]
```diff
@@ -198,13 +198,6 @@ class MediaEditForm extends Component {
     e.preventDefault();
 
     let xhr = new XMLHttpRequest();
-    xhr.upload.addEventListener('progress', (evt) => {
```
Comment (maltheism) [inline modules/media/jsx/editForm.js:201]: I don't believe the following removed code did anything visual for the user. I had the same experience when testing the media upload of a file.
